pywb/templates/head_insert.html

```diff
@@ -24,10 +24,17 @@
   wbinfo.coll = "{{ coll }}";
   wbinfo.proxy_magic = "{{ env.pywb_proxy_magic }}";
   wbinfo.static_prefix = "{{ static_prefix }}/";
+{% if env.pywb_proxy_magic %}
+  wbinfo.wombat_mode = "{{ wombat_mode }}";
```

Comment:
The template already has access to the proxy config, so I think this can probably just be simplified as:

```
wbinfo.useWombat = {{ "true" if config.proxy.use_wombat else "false" }};
wbinfo.usePreserveWorker  =  {{ "true" if config.proxy.use_preserve_worker else "false" }};
```